Marlin/src/HAL/STM32F1/MarlinSerial.h

```diff
@@ -22,13 +22,16 @@
 #pragma once
 
 #include <HardwareSerial.h>
+#include <libmaple/usart.h>
 #include <WString.h>
 
 #include "../../inc/MarlinConfigPre.h"
 #if ENABLED(EMERGENCY_PARSER)
   #include "../../feature/e_parser.h"
 #endif
 
+#define UART_IRQ_PRIO 1
```

Comment:
I suggested this priority to @rhapsodyv. This is the same priority that is used for Servo PWM generation. Both should be very fast interrupts, so this is probably fine. If we find this makes the BLTouch unstable again, we can shuffle priorities to keep Servo highest.